pipeline/email.py

```diff
@@ -0,0 +1,102 @@
+"""Script to notify users via email when the price of a product drops below a certain threshold."""
+
+from os import environ as ENV
+import logging
+
+import boto3
+from boto3.exceptions import Boto3Error
+from psycopg2.extensions import connection
+
+from connect_to_database import configure_logging, get_connection, get_cursor
+
+
+def get_ses_client() -> boto3.client:
+    """Returns the boto3 SES client to send emails with."""
+
+    return boto3.client("ses", region_name="eu-west-2",
+                        aws_access_key_id=ENV["AWS_ACCESS_KEY_ID"],
+                        aws_secret_access_key=ENV["AWS_ACCESS_SECRET_KEY"])
```

Comment:
Optional! 
```suggestion
aws_access_key = ENV.get("AWS_ACCESS_KEY_ID")
aws_secret_key = ENV.get("AWS_ACCESS_SECRET_KEY")
from_email = ENV.get("FROM_EMAIL")

if not aws_access_key or not aws_secret_key or not from_email:
    raise RuntimeError("Required environment variables for AWS or email are missing")
```

Reply:
Ooh, great idea I like that! I'll implement this now :)